README.md

```diff
@@ -0,0 +1,31 @@
+# myHPI
+
+This tool is used to manage the student representative website at https://myhpi.de (not live yet). It is a CMS based on Wagtail/Django and adds several functionalities like polls.
+
+## Development setup
+
+To set up a development version on your local machine, you need to execute the following steps:
+1. Check out repository and cd to it
+2. Set up a virtualenv for the project with Python >=3.8 and activate it
+3. Install poetry (if not already installed): `curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -`
+4. Install dependencies with `poetry install`
+5. Create env file by copying the `.env.example file to `.env`, e.g. `cp .env.example .env`
```

Comment:
is there the need to change something? e.g. I saw in #29 that there is `OIDC_RP_CLIENT_ID=id`and  `OIDC_RP_CLIENT_SECRET=secret`in the env.example?

Reply:
Yeah some things must be changed. We should add this.

Reply:
there is no need to change anything to get a functional setup. OIDC login will just not be functional, but as you'd need to register a client with the HPI OpenID provider for that it is not something i would recommend. 
Maybe we could add a hint that this is the case